lib/mage/Mage.js

```diff
@@ -725,11 +742,13 @@ Mage.prototype.boot = function (cb) {
 		}
 
 		var keys = Object.keys(apps);
-		keys.forEach(function (appName) {
-			that.emit('setup.' + appName, apps[appName]);
-		});
+		keys.forEach((appName) => this.emit('setup.' + appName, apps[appName]));
```

Comment:
If you're gonna rewrite this, I think a for-of loop would've been more readable. But no blocker.